### ISSUE TITLE
Unclear what to do after command line install

### ISSUE THREAD

**Issue by yoeriboven** [statamic/v2-docs#458]
According to the [installation docs](https://github.com/statamic/v2-docs/blob/master/content/collections/docs/installing.md) the only thing you have to do with a CL-install is run the command. It's unclear how to reach the admin section, add the license key, etc.

At step 5 for the manual install, it says something about running `installer.php` or go to `{yoursite}.{tld}/cp` but this is not mentioned at the CL-install docs.

Got it figured out after trial and error so since I'm not sure what the correct way is I can't post a PR myself.

**tl;dr: Installation documentation is unclear for command line installation.**
